packages/EasyCodingStandard/src/Console/Application.php

```diff
@@ -39,7 +39,7 @@ private function addExtraOptions(InputDefinition $inputDefinition): void
             null,
             InputOption::VALUE_REQUIRED,
             'Path to config file.',
-            getcwd() . '/easy-coding-standard.yml'
+            'easy-coding-standard.(yml|yaml)'
         ));
```

Comment:
This is actually a string, not real value, right?
In that case it can be only listed in description

Reply:
This argument here is just for documentation purposes, it's not otherwise used anywhere. But yes, this is otherwise real value. What I specified is valid shell wildcard path, support for it can be implemented later if somebody will need such thing. For now it's just documentation.

Reply:
I see, thanks :+1: